tests/unit/ena/webin_cli_handler/test_webin_cli_handler.py

```diff
@@ -0,0 +1,152 @@
+import subprocess
+import time
+import os
+
+
+timestamp = int(time.time())
+timestamp_genomes = int(time.time())
+webin_version = os.getenv("WEBIN_CLI_VERSION")
+
+class Tests:
+    def test_validate_assembly_upload(tmp_path):
+        command = [
+            "python",
+            "mgnify_pipelines_toolkit/ena/webin_cli_handler.py",
+            "-c",
+            "genome",
+            "-m",
+            "tests/fixtures/webin_cli_handler/assembly.manifest",
+            "--mode",
+            "validate",
+            "--webin-cli-jar",
+            f"webin-cli-{webin_version}.jar",
+        ]
+        result = subprocess.run(command, capture_output=True, text=True)
+        assert result.returncode == 0, f"Run failed: {result.stderr}"
+        assert "Submission validation succeeded" in result.stderr
+        assert "Submission/validation done for" in result.stderr
+
+    def test_validate_genome_upload(tmp_path):
+        command = [
+            "python",
+            "mgnify_pipelines_toolkit/ena/webin_cli_handler.py",
+            "-c",
+            "genome",
+            "-m",
+            "tests/fixtures/webin_cli_handler/genome.manifest",
+            "--mode",
+            "validate",
+            "--webin-cli-jar",
+            f"webin-cli-{webin_version}.jar",
+        ]
+        result = subprocess.run(command, capture_output=True, text=True)
+        assert result.returncode == 0, f"Run failed: {result.stderr}"
+        assert "Submission validation succeeded" in result.stderr
+        assert "Submission/validation done for" in result.stderr
+
+    def test_submit_assembly_upload_first_time_test_server(tmp_path):
+        test_manifest = "tests/fixtures/webin_cli_handler/assembly.manifest"
+        # create a new manifest with another alias for unique submission
+        with open("new_assembly.manifest", "w") as file_out, open(test_manifest, "r") as file_in:
+            for line in file_in:
+                if "ASSEMBLYNAME" in line:
+                    line = f"ASSEMBLYNAME\ttest_{timestamp}_a\n"
+                file_out.write(line)
+        command = [
+            "python",
+            "mgnify_pipelines_toolkit/ena/webin_cli_handler.py",
```

Comment:
Shouldn't we run this from as `webin_cli_handler` and not with python ... script.py?